packages/discordx/examples/button/commands/game.ts

```diff
@@ -66,11 +65,10 @@ class spcProposition {
 const defaultChoice = new spcProposition(spcChoice.Stone, "💎", "spc-stone");
 
 @Discord()
-@Bot("alexa")
 export abstract class StonePaperScissor {
```

Comment:
You should probably rename the class as well

Reply:
done